src/main/app/src/pages/HomePage/Filters.tsx

```diff
@@ -80,13 +83,34 @@ export const Filters = ({
   onKoulutuksenAlkamiskausiChange,
   koulutuksenAlkamisvuosi,
   onKoulutuksenAlkamisvuosiChange,
+  orgWhitelist,
+  onOrgWhitelistChange,
 }) => {
   const { t } = useTranslation();
 
   const tilaOptions = useTilaOptions(t);
 
   const koulutustyyppiOptions = useKoulutustyyppiOptions(t);
 
+  const parseChildOrgs = (hierarkia, selectedOrg, lang) => {
+    let flatHierarkia = flattenHierarkia(hierarkia)
+    let result = []
+      console.log('parse child orgs for lang ' + lang)
+    flatHierarkia.forEach(org => {
+        if (org && org.oid && org.oid != selectedOrg && org && org.lyhytNimi) {
```

Comment:
Tämä onnistuu vähän lyhyemmin näin:
```suggestion
        if (org?.oid != selectedOrg && org?.lyhytNimi) {
```